synapse/models/infotech.py

```diff
@@ -934,6 +956,92 @@ def getModelDefs(self):
                         'doc': 'An array of ChildOf CWE Relationships.'
                     }),
                 )),
+
+                (('it:sec:metrics', {}, (
+
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization whose security program is being measured.'}),
+
+                    ('org:name', ('ou:name', {}), {
+                        'doc': 'The organization name. Used for entity resolution.'}),
+
+                    ('window', ('ival', {}), {
+                        'doc': 'The time window used to compute the metrics.'}),
+
+                    ('alert:meantime:triage', ('duration', {}), {
+                        'doc': 'The mean time to triage for alerts generated within the time window.'}),
+
+                    ('alert:count', ('int', {}), {
+                        'doc': 'The total number of alerts generated within the time window.'}),
+
+                    ('alert:falsepos', ('int', {}), {
+                        'doc': 'The number of alerts generated within the time window that were determined to be false positives.'}),
+
+                    ('assets:hosts', ('int', {}), {
+                        'doc': 'The total number of hosts within scope for the information security program.'}),
+
+                    ('assets:users', ('int', {}), {
+                        'doc': 'The total number of users within scope for the information security program.'}),
+
+                    ('assets:vulns:existing', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities being tracked at the beginning of the window.'}),
+
+                    ('assets:vulns:discovered', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities discovered during the window.'}),
+
+                    ('assets:vulns:mitigated', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities mitigated during the window.'}),
+
+                    ('assets:vulns:meantime:mitigate', ('duration', {}), {
+                        'doc': 'The mean time to mitigate for vulnerable assets mitigated during the time window.'}),
+
+                )),
+
+                ('it:sec:vuln:scan', {}, (
+
+                    ('time', ('time', {}), {
+                        'doc': 'The time that the scan was started.'}),
+
+                    ('desc', ('str', {}), {
```

Comment:
had we talked about capturing what is being scanned (besides just in the description)? If not, would we want to include something like an :assets:scanned array property so that people can capture what assets were scanned (and not just those that produced a hit)?

Reply:
This could be in the millions of hosts and/or ip:port combinations. We could discuss a scope edge or something, but i'd like to kick that can down the road for now. :)